src/main/java/com/nextstep/calculator/domain/Numbers.java

```diff
@@ -0,0 +1,43 @@
+package com.nextstep.calculator.domain;
+
+import com.nextstep.calculator.domain.exceptions.InvalidNumbersException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Numbers {
+    private final List<Number> numbers;
+
+    public Numbers(final List<Number> numbers) {
+        validate(numbers);
+        this.numbers = new ArrayList<>(numbers);
+    }
+
+    public void validate(final List<Number> numbers) {
+        if (numbers == null) {
+            throw new InvalidNumbersException("null로 객체를 생성할 수 없습니다.");
+        }
+        if (numbers.size() == 0) {
```

Comment:
사소하지만 아래와 같이 사용하면 가독성을 향상시킬 수 있을 것 같아요~
```suggestion
        if (numbers.isEmpty()) {
```